scrapy/msg_que/demo_queue.py

```diff
@@ -0,0 +1,60 @@
+import scrapy
+import redis
+import random
+
+
+class redis_spider(scrapy.Spider):
```

Comment:
A queue class should not inherit from `Spider`. They do not need to inherit from any class. And they should be enabled through settings: https://docs.scrapy.org/en/latest/topics/settings.html#scheduler-disk-queue

See @whalebot-helmsman’s latest message on the topic here: https://github.com/scrapy/scrapy/issues/4326#issuecomment-591332112